### PR TITLE
Svg

### PR DESCRIPTION
**Pull request by AaronGe88inTHU**
Add Text & G Tags

### CODE REVIEW

File: Sources/SwiftSvg/Svg+G.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//  
+//
+//  Created by Aaron Ge on 2022/4/26.
+//
+
+import SwiftSvg
+
+
+extension Svg{
+    open class G: Tag{
```
Comment (tib) [inline Sources/SwiftSvg/Svg+G.swift:12]: Can you move this out from the Svg extension & follow the standard conventions? See Line, Rect, etc. tags. Also could you provide docs & test for these new tags? Thank you.

File: Sources/SwiftSvg/Svg+Text.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//
+//  Svg+Text.swift
+//  
+//
+//  Created by Aaron Ge on 2022/4/8.
+//
+
+extension Svg{
```
Comment (tib) [inline Sources/SwiftSvg/Svg+Text.swift:8]: Could you move out the text from the Svg scope? The SwiftSvg library already provides the module namespace so we don't need to place Svg related objects under the Svg extension. Also please provide some basic docs & unit tests if possible. Thank you. 👍

**Review by tib — CHANGES_REQUESTED** [review]
Thanks for the contribution, please check my comments to move forward with this PR.

File: Sources/SwiftSvg/Svg+Text.swift (hunk 2 above)
Comment (AaronGe88inTHU) [reply Sources/SwiftSvg/Svg+Text.swift]: move Text Tag out from Svg extension

File: Sources/SwiftSvg/Svg+G.swift (hunk 1 above)
Comment (AaronGe88inTHU) [reply Sources/SwiftSvg/Svg+G.swift]: move G Tag out from Svg extension

**Comment by AaronGe88inTHU** [comment]
move Text & G Tag out from Svg extension 
simple test and doc added

**Pull request closed without merging**